backend/typescript/types.ts

```diff
@@ -1,5 +1,7 @@
 export type Role = "User" | "Admin";
 
+export type DonorRole = "LocalBusiness" | "IndividualDonor";
```

Comment:
nit but can we make this 
```suggestion
export type DonorRole = "Local Business" | "Individual Donor";
```